src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -35,31 +54,30 @@ object Cluster {
     labels = clusterRequest.labels)
 }
 
-case class Cluster(clusterName: String,
+case class Cluster(clusterName: ClusterName,
                    googleId: UUID,
                    googleProject: GoogleProject,
                    googleServiceAccount: GoogleServiceAccount,
                    googleBucket: GoogleBucket,
-                   operationName: String,
+                   operationName: OperationName,
                    status: ClusterStatus,
-                   hostIp: Option[String],
+                   hostIp: Option[IP],
                    createdDate: Instant,
                    destroyedDate: Option[Instant],
-                   labels: Map[String, String])
+                   labels: LabelMap)
 
 case class ClusterRequest(bucketPath: GoogleBucket,
-                          serviceAccount: String,
-                          labels: Map[String, String])
+                          serviceAccount: GoogleServiceAccount,
+                          labels: LabelMap)
 
-case class ClusterResponse(clusterName: String,
+case class ClusterResponse(clusterName: ClusterName,
                            googleProject: GoogleProject,
-                           googleId: String,
+                           googleId: UUID,
                            status: String,
```

Comment:
We don't use `status` or `description` but they're present in Google's response JSON.